cfg/Instructions.h

```diff
@@ -167,6 +167,21 @@ class LoadArg final : public Instruction {
 };
 CheckSize(LoadArg, 24, 8);
 
+class ArgPresent final : public Instruction {
+public:
+    core::SymbolRef method;
+    u2 argId;
+
+    ArgPresent(core::SymbolRef method, int argId) : method(method), argId(argId) {
```

Comment:
You're initializing a `u2` from an `int`?

Reply:
Hmm, that is questionable. It's the same for `LoadArg` though, should I just update them both?

Reply:
I think that this wasn't caught in either case because both `argId` values are initialized as `argId(argId)` instead of `argId{argId}`.

Reply:
Yes, I think we should change both instructions.